velocity/src/main/java/net/william278/huskchat/velocity/player/VelocityPlayer.java

```diff
@@ -60,11 +60,6 @@ public boolean hasPermission(String permission) {
         return player.hasPermission(permission);
     }
 
-    @Override
-    public void passThroughChat(String message) {
-        player.spoofChatInput(message);
```

Comment:
This still works in 1.19, actually. I'm not sure we want to degrade functionality on Velocity for our Bungee users to be honest.